common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -731,4 +771,90 @@ public void nRingsUnsupported() {
         assertEquals(expected, e.getMessage());
     }
 
+    @Test
+    public void translateEmptyObjectNoDeltaZ() {
```

Comment:
Please change the test cases to use different values for deltaX and deltaY (currently they are all `1`) or add a test case that uses different values for detlaX and deltaY.

This will test if x and y are handled correctly.

Reply:
Done